xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/src/main/resources/templates/mentions/mention.vm

```diff
@@ -83,7 +83,8 @@
             #else
                 #set ($descriptionKey = 'mentions.event.mention.description')
             #end
-            #set ($description = "$services.localization.render($descriptionKey) <a href='$eventParams.getDocumentURL()'>$eventParams.getDocument().getTitle()</a>")
+            #set ($tdoc = $xwiki.getDocument($eventParams.getDocument().getDocumentReference()).getTranslatedDocument())
+            #set ($description = "$services.localization.render($descriptionKey) <a href='$eventParams.getDocumentURL()'>$tdoc.getRenderedTitle('xhtml/1.0')</a>")
```

Comment:
@manuelleduc is there any reason to not move the `<a>` on a new line. It looks like this line exceeds the 120 chars limit. 
Also, I don't understand why the indentation uses 4 spaces instead of 2 in this vm file. Am I missing something? 
And, not that important, you can always use
` #set ($tdoc = $xwiki.getDocument($eventParams.document.documentReference).translatedDocument)` in velocity if it helps to fit the code in one line.

Reply:
The two code style violations are my fault, thanks for spotting them, I'll create another PR to fix them.